custom_components/pioneer_async/media_player.py

```diff
@@ -330,10 +330,18 @@ def supported_features(self) -> MediaPlayerEntityFeature:
         ):
             features |= MediaPlayerEntityFeature.SELECT_SOUND_MODE
 
-        ## Enable prev/next track if tuner enabled
-        if pioneer.source.get(self.zone) == SOURCE_TUNER:
-            features |= MediaPlayerEntityFeature.PREVIOUS_TRACK
-            features |= MediaPlayerEntityFeature.NEXT_TRACK
+        control_commands = pioneer.get_supported_media_controls(self.zone)
```

Comment:
This section needs to be conditional on the currently active source supporting media controls. As it stands, this code removes the capability of using previous/next track when the tuner is active.

Reply:
> This section needs to be conditional on the currently active source supporting media controls.

Isn't this what `pioneer.get_supported_media_controls()` should return here? So if the currently active source does not support media controls, it should return nothing, otherwise it should return whatever is currently supported?

> As it stands, this code removes the capability of using previous/next track when the tuner is active.

Is this a bug in `PioneerAVR.get_supported_media_controls()` then? At least for a `NETWORK` source it seems to work fine for me.

Reply:
Hmmm, you could be right. I didn't write the backend code for this, so am less familiar with what its intent is. I had assumed that it was used only for MHL and network sources (I don't use any of these in my setup, which is why the frontend never supported it) but going through the backend code I see that it is supposed to support a tuner source as well. Let me test the PR with the tuner. Thanks.

Reply:
Tested working for `previous` and `next` for a TUNER source as well.